Can can
bool eps_enable_status
uint8 eps_control_board_status
uint8 eps_control_status
bool eps_user_can_err
bool eps_err
bool eps_vehicle_can_err
bool eps_sas_err
bool override_ignore_status
bool override_status
int16 steer_angle
float64 steer_drive_torque
float64 steer_out_torque
uint8 eps_alive_count